### PR TITLE
fix(design): adjustments for Nextcloud 30 - v3

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Ref #12800

### 🖼️ Screenshots / Screencasts

🏚️ Before | 🏡 After
-- | --
image | image
image | image
image | image
image | image
image | image

## Conversation and composing

* [ ] Top bar title is cut off - dropped for now
* [x] Date marker "Today, July 23, 2024" border radius should be reduced to make it similar to the button

## List view

* [x] Margin on the left of list item is too much and on the right is too little --> it should nicely aligned with the left of the search bar and right aligned with the new conversation button (upstream issue?)

## Sidebar

* [x] Media tab: thumbnails are top left aligned their containers

## Calling

* [x] Media settings:
  - [x] border radius on bg should be consistent
  - [x] "Test devices" button should be left aligned with select components

### 🏁 Checklist

- [ ] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences
- [ ] 🖥️ Tested with Desktop client or should not be risky for it 
- [x] 🖌️ Design was reviewed, approved or inspired by the design team
- [ ] ⛑️ Tests are included or not possible
- [ ] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required

### CODE REVIEW

File: src/components/LeftSidebar/LeftSidebar.vue [hunk 1]
```diff
@@ -1086,7 +1086,9 @@ export default {
 	padding: 0 !important;
 }
 
+// Overwrite NcListItem styles to reduce a component height
 :deep(.list-item) {
+	padding: var(--default-grid-baseline) calc(var(--default-grid-baseline) * 2);
```
Comment (ShGKme) [inline src/components/LeftSidebar/LeftSidebar.vue:1091]: Issue on upstream?

File: src/components/MediaSettings/MediaDevicesSelector.vue
Comment (ShGKme) [inline src/components/MediaSettings/MediaDevicesSelector.vue:1]: Let's also make the refresh button `tertiary` here? Same problem as on AdminSettings page

File: src/components/LeftSidebar/LeftSidebar.vue (hunk 1 above)
Comment (Antreesy) [reply src/components/LeftSidebar/LeftSidebar.vue:1091]: I'm not sure if it wasn't an intended design change for compatibility, but can upstream of course

File: src/components/TopBar/TopBar.vue [hunk 2]
```diff
@@ -347,6 +347,7 @@ export default {
 		width: 100%;
 		overflow: hidden;
 		min-height: var(--default-clickable-area);
+		font-size: 13px;
```
Comment (ShGKme) [inline src/components/TopBar/TopBar.vue:350]: This makes the text smaller than default even when there is only a title without a description.
And there is some additional space if we reduce the padding.

@jancborchardt What do you think is better here?

Also, we can stop showing the description here and show it in the sidebar. It is one line for a long (potentially rich) text anyway.

File: src/components/LeftSidebar/LeftSidebar.vue (hunk 1 above)
Comment (ShGKme) [reply src/components/LeftSidebar/LeftSidebar.vue:1091]: I'd say, if Talk needs non-standard list item design now, then something is wrong.

File: src/components/TopBar/TopBar.vue (hunk 2 above)
Comment (Antreesy) [reply src/components/TopBar/TopBar.vue:350]: Reverted changes, will provide this suggestion as a separate PR

**Review by ShGKme — APPROVED** [review]
Looks good everywhere, fixes all the issues

**Pull request merged**